### PR TITLE
Update to Beta 0 with webpack

### PR DESCRIPTION
**Pull request by robwormald**

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
@robwormald LGTM generally speaking, but I don't see where are you coping zone to lib

**Comment by robwormald** [comment]
i'm not - its committed in here - which sort of sucks, but is a temporary solution (rather than firing up a whole new gulp setup just to copy one file).

**Comment by pkozlowski-opensource** [comment]
can't we just add `cp` or sth in the `npm start`?

**Comment by robwormald** [comment]
windows users. i'll do it with a node task. hold plz...

**Comment by Foxandxss** [comment]
Or you can do what @gdi2290 does on his repo. Have one file to import every dependency and webpack will handle everything for you. Not needed to copy stuff by hand (or with scripts).

To be honest, I don't like any of the proposed ways, but I don't we have a right way for managing angular 2 deps with webpack as today.

**Comment by robwormald** [comment]
@Foxandxss the point here is we want to avoid bringing in Zone.js as a require/import - otherwise this would be pretty straightforward.

**Comment by robwormald** [comment]
merging this. should work cross platform, any issues please, well, open an issue.

File: README.md [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
 - Clone or fork this repository
 - Make sure you have [node.js](https://nodejs.org/) installed
 - run `npm install` to install dependencies
 - run `npm start` to fire up dev server
-- open browser to `http://localhost:8080`
\ No newline at end of file
+- open browser to `http://localhost:8080`
+
+### Known Issues
+- angular2-polyfills.js is currently being handled manually.
```
Comment (pkozlowski-opensource) [inline README.md]: I guess it now means "outside of WebPAck"

**Pull request merged**